<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>452</width>
         <height>386</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labPicture">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDeleteItem"/>
   <addaction name="actScanItems"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actZoomFitH"/>
   <addaction name="separator"/>
   <addaction name="actDockVisible"/>
   <addaction name="actDockFloat"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片文件</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icons/15.ico</normaloff>:/images/icons/15.ico</iconset>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/icons/folder1.bmp</normaloff>:/images/icons/folder1.bmp</iconset>
         </property>
         <item>
          <property name="text">
           <string>图片节点</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组2</string>
         </property>
         <item>
          <property name="text">
           <string>图片2</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/open3.bmp</normaloff>:/images/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/824.bmp</normaloff>:/images/icons/824.bmp</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/418.bmp</normaloff>:/images/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/416.bmp</normaloff>:/images/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/414.bmp</normaloff>:/images/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/424.bmp</normaloff>:/images/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/delete1.bmp</normaloff>:/images/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/426.bmp</normaloff>:/images/icons/426.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/fold.bmp</normaloff>:/images/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/414.bmp</normaloff>:/images/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/814.bmp</normaloff>:/images/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
